lms/templates/courseware/enroll_banner.html

```diff
@@ -0,0 +1,25 @@
+<%page args="show_enroll_banner, url_to_enroll"/>
```

Comment:
The enrollment bar present in the "info.html" is added also to the "university_id.html" and therefore I moved it ti a separate file and did an include for it in the mentioned htmls above.